deployment/ccip/changeset/cs_configure_token_pools.go

```diff
@@ -0,0 +1,320 @@
+package changeset
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	commoncs "github.com/smartcontractkit/chainlink/deployment/common/changeset"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/utils"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/token_admin_registry"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/token_pool"
+)
+
+var _ deployment.ChangeSet[ConfigureTokenPoolContractsConfig] = ConfigureTokenPoolContractsChangeset
+
+// RateLimiterConfig defines the inbound and outbound rate limits for a remote chain.
+type RateLimiterConfig struct {
+	// Inbound is the rate limiter config for inbound transfers from a remote chain.
+	Inbound token_pool.RateLimiterConfig
+	// Outbound is the rate limiter config for outbound transfers to a remote chain.
+	Outbound token_pool.RateLimiterConfig
+}
+
+// validateRateLimterConfig validates rate and capacity in accordance with on-chain code.
+// see https://github.com/smartcontractkit/ccip/blob/ccip-develop/contracts/src/v0.8/ccip/libraries/RateLimiter.sol.
```

Comment:
Nit: lets link to something in this repo, since `ccip` is basically all but deprecated at this point.

Reply:
https://github.com/smartcontractkit/chainlink/pull/15877/commits/afc13ca92a4c18ced03712680bcab6df42c0f512